input/docs/issue-providers/gitrepository/features.md

```diff
@@ -0,0 +1,36 @@
+---
+Order: 20
+Title: Features
+Description: Features of the Cake.Issues.GitRepository addin.
+---
+The [Cake.Issues.GitRepository addin] provides the following features.
+
+# Basic features
+
+* Checks if binary files are tracked by Git LFS.
+
+# Supported comment formats
+
+|                                                                    | Comment format                 | Remarks                        |
+|--------------------------------------------------------------------|--------------------------------|--------------------------------|
+| <span class="glyphicon glyphicon-ok" style="color:green"></span>   | `IssueCommentFormat.PlainText` |                                |
```

Comment:
why not linking the enums to [IssueCommentFormat](https://cakebuild.net/api/Cake.Issues/IssueCommentFormat/) docu?